map/src/infoblock/components/track/GeneralInfo.jsx

```diff
@@ -331,7 +334,7 @@ export default function GeneralInfo({ width, setOpenDescDialog }) {
                         </ListItemIcon>
                         <ListItemText>
                             <Typography sx={{ ml: 1 }} variant="body2" noWrap>
-                                {`Elevation SRTM (min/avg/max): ${elevationSRTM}`}
+                                {`Elevation SRTM: ${elevationSRTM}`}
```

Comment:
Why did you delete (min/avg/max)?

Reply:
This is temporarily change to fit Ele-SRTM string into 1 line. Let's improve it later when testing mobile version.